core/object/object.cpp

```diff
@@ -1483,10 +1483,13 @@ String Object::tr(const StringName &p_message, const StringName &p_context) cons
 	}
 
 	if (Engine::get_singleton()->is_editor_hint()) {
-		return TranslationServer::get_singleton()->tool_translate(p_message, p_context);
-	} else {
-		return TranslationServer::get_singleton()->translate(p_message, p_context);
+		String tr_msg = TranslationServer::get_singleton()->extractable_translate(p_message, p_context);
```

Comment:
To be clear, is it intentional that `tr_msg` isn't returned?

Reply:
Oops! Fixed.